### PR TITLE
feat: setup cluster resources required for tekton dashboard

### PR DESCRIPTION
**Pull request by harshad16**
setup cluster resources required for tekton dashboard
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

Tekton dashboard is a one-stop place to view all the openshift pipelines running. 
https://github.com/tektoncd/dashboard/blob/main/docs/install.md

### CODE REVIEW

File: cluster-scope/base/rbac.authorization.k8s.io/clusterrolebindings/tekton-dashboard/clusterrolebinding.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
```
Comment (HumairAK) [inline cluster-scope/base/rbac.authorization.k8s.io/clusterrolebindings/tekton-dashboard/clusterrolebinding.yaml:1]: Can you break these up as per the format described here: https://github.com/operate-first/apps/tree/master/cluster-scope#application-folder-structure

Same with the clusterroles. 


(seems tedious but this is the current structure, we're debating the cons of this format [here](https://github.com/operate-first/SRE/issues/369) if you're interested )

Comment (harshad16) [reply cluster-scope/base/rbac.authorization.k8s.io/clusterrolebindings/tekton-dashboard/clusterrolebinding.yaml]: ack, I have updated with more breakdowns now
does it fit right ?

**Review by 4n4nd — COMMENTED** [review]
/lgtm

**Pull request merged**